src/_bentoml_impl/loader.py

```diff
@@ -70,20 +71,23 @@ def normalize_identifier(
 
         # TODO(jiang): bento store configs are sdk configs, should be moved to sdk in the future
         from bentoml._internal.configuration.containers import BentoMLContainer
-        from bentoml.exceptions import NotFound
+        from bentoml.exceptions import BentoMLException
 
         bento_store = BentoMLContainer.bento_store.get()
 
         try:
             bento = bento_store.get(service_identifier)
-        except NotFound:
+        except BentoMLException:
             # a python import str
-            return normalize_package(service_identifier), pathlib.Path(
-                working_dir if working_dir is not None else "."
-            )
+            module_spec, _, service_path = service_identifier.partition(":")
+            module_path = pathlib.Path(module_spec)
+            if not module_path.suffix:
+                return service_identifier, pathlib.Path(working_dir or ".")
+            if not module_path.is_absolute():
+                module_path = pathlib.Path(working_dir or ".").joinpath(module_path)
+            return f"{module_path.stem}:{service_path}", module_path.parent
```

Comment:
Shall we return CWD as the bento path?

Reply:
bento_path here is to tell `import_service` where to import the service, so it makes sense to return the parent dir of it.

This is to handle the case where `bento_identifier` is a path.

Reply:
bento_path is used to be the working directory of sub-processes

Reply:
So `normalize_identifier` is to return two values, import string and bento_path. For a bento identifier as absolute path. What shall it return? `normalize_identifier("/path/to/package.py:Inference")`

Reply:
@frostming

Reply:
You just said this case happens when user call `service.serve_http()`, thus I think we should inherit the working directory, which is '.'
Will that work?
@frostming

Reply:
but the first return value will be directly feed to `import_service`. An absolute path isn't a valid import string right?